package.json

```diff
@@ -3,7 +3,7 @@
 	"private": true,
 	"version": "1.0.0",
 	"description": "",
-	"main": "dist/app.js",
+	"main": "src/register.ts",
```

Comment:
Why did you choose to use the one-off script as the main entry point?

Reply:
was working on trying to get my gripes fixed. Will change